packages/reports/tests/acceptance/report-test.js

```diff
@@ -1061,6 +1105,16 @@ module('Acceptance | Navi Report', function(hooks) {
     await visit('/reports');
     await visit('/reports/new');
     await clickItem('metric', 'Ad Clicks');
+    await clickItem('dimension', 'Date Time');
+    await clickItemFilter('dimension', 'Date Time');
+    await click($('.filter-builder__operator:contains(Since) .filter-builder__select-trigger')[0]);
+    await click($('li.ember-power-select-option:contains(Between)')[0]);
```

Comment:
Use this going forward
```suggestion
    await selectChoose('.filter-builder__select-trigger', 'Between');
```